applications/nrf_desktop/doc/leds.rst

```diff
@@ -0,0 +1,43 @@
+.. leds:
+
+leds module
+###########
+
+``leds`` module is responsible for controlling LED diodes (setting LED effect in reaction to ``led_event``).
+
+
+Configuration
+*************
+
+``leds`` module is based on Zephyr PWM driver.
```

Comment:
For setting brightness leds uses PWM driver.

Reply:
done